pkg/kudoctl/cmd/package.go

```diff
@@ -0,0 +1,73 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/kudobuilder/kudo/pkg/kudoctl/bundle"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+const (
+	example = `
+		The package argument must be a directory which contains the operator definition files.  The package command will create a tgz file containing the operator.
+
+		# package zookeeper (where zookeeper is a folder in the current directory)
+		kubectl kudo package zookeeper
+
+		# Specify a destination folder other than current working directory
+		kubectl kudo package ../operators/repository/zookeeper/operator/ --destination=out-folder`
+)
+
+type packageCmd struct {
+	path        string
+	destination string
+	overwrite   bool
+	out         io.Writer
+	fs          afero.Fs
+}
+
+// newPackageCmd creates an operator bundle.  fs is the file system, out is stdout for CLI
+func newPackageCmd(fs afero.Fs, out io.Writer) *cobra.Command {
+
+	b := &packageCmd{out: out, fs: fs}
+	cmd := &cobra.Command{
+		Use:     "package <operator_dir>",
+		Short:   "Package an official KUDO operator.",
+		Long:    `Package a KUDO operator from local filesystem.`,
```

Comment:
should at least some of this said that package == create tarball? I mean package can mean various things while this is creating tarballs...